### PR TITLE
More changes to translations links

### PR DESCRIPTION
**Pull request by joaquinferrero**
Unified titles (Japanese).
Remove Italian Source Forge project link because is referred into first Italian link (and corrected).
Ordered by last update time.

### CODE REVIEW

File: docs/www/docs.html [hunk 1]
```diff
@@ -58,26 +58,27 @@
...
+            <a href="http://www.perl.it//documenti/pod2it/">Italian</a>
         </li>
         <li>
-            <a href="https://metacpan.org/release/POD2-ES">Spanish</a>
+            <a href="http://perl.enstimac.fr/DocFr.html">French</a>
         </li>
         <li>
-            <a href="https://metacpan.org/release/POD2-RU">Russian</a>
-</li>    </ul>
+            <a href="https://metacpan.org/release/POD2-LT">Lithuanian</a>
```
Comment (rspier) [inline docs/www/docs.html]: Is this worth including here?  It has only a tiny amount of coverage. https://metacpan.org/source/KUDARASP/POD2-LT-0.02/lib/POD2/LT

File: docs/www/docs.html [hunk 2]
```diff
@@ -58,26 +58,27 @@
...
-            <a href="http://www.perl.it/documenti/index.html">Italian</a> (Also <a href="http://pod2it.sourceforge.net/">pod2it</a>)
+            <a href="https://metacpan.org/release/POD2-ES">Spanish</a>
         </li>
         <li>
-            <a href="http://perl.enstimac.fr/DocFr.html">French</a>
+            <a href="https://metacpan.org/release/POD2-RU">Russian</a>
         </li>
         <li>
-            <a href="https://metacpan.org/release/POD2::PT_BR">Brazilian portuguese</a>
+            <a href="https://metacpan.org/release/POD2-PT_BR">Brazilian portuguese</a>
```
Comment (rspier) [inline docs/www/docs.html]: Coverage on this is similar to on the Lithuanian translation.

File: docs/www/docs.html [hunk 3]
```diff
@@ -58,26 +58,27 @@
     <h3>Translations</h3>
     <ul class="list-unstyled">
         <li>
```
Comment (rspier) [inline docs/www/docs.html]: What is the new sort order?  It's not alphabetical.

**Comment by rspier** [comment]
p.s. Thanks!

File: docs/www/docs.html (hunk 3 above)
Comment (rspier) [inline docs/www/docs.html]: I see the comment.  Last Update time.  That's not obvious (or terribly useful.)  Alphabetical is going to be the simplest and most obvious.

Comment (joaquinferrero) [inline docs/www/docs.html]: Ok.

File: docs/www/docs.html (hunk 1 above)
Comment (joaquinferrero) [inline docs/www/docs.html]: Yes, you are right.

**Comment by rspier** [comment]
If you're going to make a followup change based on comments you can just modify the pull request.  GitHub has instructions somewhere for how to do that.

**Pull request closed without merging**